packages/cdk/lib/stacks/engines/cromwell-engine-construct.ts

```diff
@@ -99,8 +106,8 @@ export class CromwellEngineConstruct extends EngineConstruct {
     });
     const definition = new FargateTaskDefinition(this, "EngineTaskDef", {
       taskRole: this.engineRole,
-      cpu: serviceContainer.cpu,
-      memoryLimitMiB: serviceContainer.memoryLimitMiB,
+      cpu: (params.vCpus ? params.vCpus * 1024 : undefined) || serviceContainer.cpu,
```

Comment:
What is the meaning of this `1024` magic number? can you use a constant with a meaningful name instead? I think it will improve readability and will serve self-documenting purpose.